stylesheets/bootstrap.min.css

```diff
@@ -1257,7 +1257,7 @@ hr {
     line-height:1.4
 }
 
-@media (min-width:768px) {
+@media (min-width:1300px) {
```

Comment:
The fact that you've edited the bootstrap source file makes me raise my eyebrow.